composer.json

```diff
@@ -16,6 +16,10 @@
     ],
     "require": {
         "php": "^7.1",
+        "ext-json": "*",
+        "ext-libxml": "*",
+        "ext-simplexml": "*",
+        "ext-dom": "*",
```

Comment:
These should be in `require-dev` section, as should be installed only if used, but they are not required to use serializer (none of them)

Reply:
👍